mw-webapp/src/pages/mainPage/MainPage.tsx

```diff
@@ -11,7 +12,9 @@ export const MainPage = () => {
       <h1 className={styles.title}>
         Hiii, Student!
       </h1>
-      <Table />
+      <WrapperGetDataTable>
+        <Table context={TableUsersContext} />
```

Comment:
are you sure we should use context as prop? IS it standard approach?

Reply:
Maybe no, if we don’t use context, you can make a wrapper from another component and inside the table

Reply:
I think just ordinary data prop will be enough

Reply:
done